job-service-db/src/main/resources/db/migration/V2__job_service.sql

```diff
@@ -0,0 +1,54 @@
+--
+-- Copyright 2016-2021 Micro Focus or one of its affiliates.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+/*
+ *
+ *  This script allows the migration from a Liquibase configuration onto Flyway
+ *
+ *  It will check if any databasechangelog (Liquibase specific table) exists.
+ *  If so, it will check if the latest md5sum is present. In case it isn't, the migration aborts.
+ *  Otherwise, the Liquibase specific tables get dropped (databasechangelog, databasechangeloglock)
+ *
+ */
+DO
+$$
+    BEGIN
+
+        IF EXISTS
+            (
+            -- Checks if liquibase table exists
+                SELECT table_name
```

Comment:
You're only interested in whether or not the row exists.
```suggestion
                SELECT NULL
```

Reply:
Done